index.html

```diff
@@ -1270,11 +1270,11 @@ <h2>Extensibility</h2>
 together.
         </li>
       <li>
-Representations MAY define other extensibility mechanisms including methods for
-decentralized extensions. Such extension mechanisms SHOULD support lossless
-conversion into any other conformant representation. Extension mechanisms for a
-representation SHOULD define a mapping of all properties and representation
-syntax into the <a href="#data-model">data model</a> and its type system.
+Representations MAY define other extensibility mechanisms ncluding ones that do
+not require the use of the DID Core Registries. Such extension mechanisms SHOULD
+support lossless conversion into any other conformant representation. Extension
+mechanisms for a representation SHOULD define a mapping of all properties and
+representation syntax into the abstract data model and its type system.
```

Comment:
should `data model` still be a link?

Reply:
Yup, I'll go back in and fix this. Thanks!

Reply:
Addressed, thanks for the review.